testdata/p4_16_samples_outputs/pna-issue3041.p4.spec

```diff
@@ -42,10 +40,12 @@ struct a2_arg_t {
 struct main_metadata_t {
 	bit<32> pna_main_input_metadata_input_port
 	bit<32> pna_main_output_metadata_output_port
-	bit<32> MainParserT_parser_tmp
+	bit<8> MainParserT_parser_tmp
```

Comment:
Why is changed? Is this expected?

Reply:
This is expected. MainParserT_parser_tmp is used for processing version=ihl field since it is a 8-bit field